v63004/u_inref/gtm8891.csh

```diff
@@ -0,0 +1,17 @@
+
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# all rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license. If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+
+echo "#Running gtm8891 test"
+$gtm_dist/mumps -run gtm8891
+$gtm_dist/mumps -run %XCMD 'halt'
```

Comment:
Line 17 is not necessary. The mumps -run done in line 16 returns back to the shell so no need to start a new mumps process which only does a halt.